tests/integration-tests/tests/networking/test_cluster_networking.py

```diff
@@ -9,18 +9,100 @@
 # or in the "LICENSE.txt" file accompanying this file.
 # This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
 # See the License for the specific language governing permissions and limitations under the License.
+import logging
+import time
+
 import pytest
+import utils
 from assertpy import assert_that
+from cfn_stacks_factory import CfnStack
+from remote_command_executor import RemoteCommandExecutor
+from troposphere import GetAtt, Output, Ref, Template, ec2
 from utils import get_compute_nodes_instance_ids
 
+from tests.common.schedulers_common import get_scheduler_commands
+from tests.common.utils import retrieve_latest_ami
+from tests.storage.test_fsx_lustre import (
+    get_fsx_fs_id,
+    test_fsx_lustre_correctly_mounted,
+    test_fsx_lustre_correctly_shared,
+)
+
 
 @pytest.mark.dimensions("us-west-2", "c5.xlarge", "alinux2", "slurm")
 @pytest.mark.dimensions("eu-west-2", "c5.xlarge", "centos7", "slurm")
 @pytest.mark.usefixtures("os", "scheduler", "instance")
-def test_cluster_in_private_subnet(region, pcluster_config_reader, clusters_factory):
+def test_cluster_in_private_subnet(region, os, scheduler, pcluster_config_reader, clusters_factory, bastion_factory):
     # This test just creates a cluster in the private subnet and just checks that no failures occur
-    cluster_config = pcluster_config_reader()
+    fsx_mount_dir = "/fsx_mount"
+    cluster_config = pcluster_config_reader(fsx_mount_dir=fsx_mount_dir)
     cluster = clusters_factory(cluster_config)
     assert_that(cluster).is_not_none()
 
     assert_that(len(get_compute_nodes_instance_ids(cluster.cfn_name, region))).is_equal_to(1)
+    _test_fsx_in_private_subnet(cluster, os, region, scheduler, fsx_mount_dir, bastion_factory)
+
+
+def _test_fsx_in_private_subnet(cluster, os, region, scheduler, fsx_mount_dir, bastion_factory):
+    """Test FSx can be mounted in private subnet."""
+    bastion_ip = bastion_factory()
+    logging.info("Sleeping for 60 sec to wait for bastion ssh to become ready.")
```

Comment:
Is there a way to check bastion/ec2 instance status rather than waiting 60 seconds?